### ISSUE TITLE
Auto-play setting sorely missing on Android 

### ISSUE THREAD

**Issue by bumjrah** [brave/brave-browser#14142]
## Description 

I know probably previous bugs have been submitted in this regard, but the information I've come across on this is either outdated or scattered and documented elsewhere.

The 1 platform (mobile version) where a toggle for disabling auto-play is mostly needed is conspicuously missing. Yet desktop version has this. 

I would think the priority for enabling/disabling auto-play would be better placed on mobile rather than desktop. Is there something am missing on this peculiar omission? I also understand default chrome/Chromium as well omits auto-play control on the Android versions. 

This flaw drives me nuts to say the least. Apart from burning through my mobile data, I don't get the chance to block the unneeded video content from auto playing itself. 

Would appreciate any pointers as to why this conspiracy exists. Or at least any development or future plans regarding this specific feature.

**Comment by srirambv** [comment]
I don't think it is possible on Android to stop auto-play unless we deviate from Chromium and add that freature.

cc: @anthonypkeane @samartnik

**Comment by bsclifton** [comment]
related / duplicate of https://github.com/brave/brave-browser/issues/13343

**Comment by mattspew** [comment]
Blocking auto-play is essential!

I can't believe it's not implemented in a browser that claims to be about privacy and data saving.

**Comment by AllamanB** [comment]
Ok i post this here again since brave team seems to prefer solving issues by closing them.

you're really taking your time with this it's ridiculous to see how brave developers don't pay any attention to one of thier most important platforms and keep losing users this function has been added to bromite by hobbist developers and it works perfectly fine.
I suggest to take a look at thier repo and add this must-have feature to brave:
https://github.com/bromite/bromite/blob/2251acbca8218619186c410d91f95b553be989d5/build/patches/Revert-the-removal-of-an-option-to-block-autoplay.patch

https://github.com/bromite/bromite/blob/2251acbca8218619186c410d91f95b553be989d5/build/patches/Disable-unified-autoplay-feature.patch

BTW please don't make excuses that you want to use a better way to block autoplay this is an important feature that you couldn't find a way to add for a long time so just do what they did.

**Comment by Spondulicks** [comment]
This is an essential feature for a web browser firefox never remove this feature, yet chromium did but bromite devs reverted thier changes while remaining on top of chromium updates. Currently auto play wastes a lot of data and it's very annoying, it should be your top priority.

**Comment by bsclifton** [comment]
Great job @wchen342 😄 Added missing milestone; do you have a test plan that QA can use?

**Comment by wchen342** [comment]
I have updated the PR with a test plan.

**Comment by stephendonner** [comment]
Verified `PASSED` using Brave Beta `1.32.94, Chromium 95.0.4638.69` on a Google Pixel XL, Android 9, using the testplan from https://github.com/brave/brave-core/pull/10120.

example | example | example | example | example
---------|----------|---------|----------|---------
![Screenshot_20211108-103000](https://user-images.githubusercontent.com/387249/140798997-031ad568-e33c-4824-a113-33ab73cd7685.png) | ![Screenshot_20211108-102318](https://user-images.githubusercontent.com/387249/140799020-33a3be7d-8f48-4dbe-bdf6-648769d015d6.png) | ![Screenshot_20211108-102601](https://user-images.githubusercontent.com/387249/140799051-d2b3fdf2-9c8c-4864-ad02-82154505c06f.png) | ![Screenshot_20211108-102509](https://user-images.githubusercontent.com/387249/140799218-9f215ec6-a727-49ac-b5e0-0dcdf57bdc34.png) |  ![Screenshot_20211108-102550](https://user-images.githubusercontent.com/387249/140799074-372449b3-6830-4177-a3a3-4f5657a06ad7.png)
